trashtalk/apps/accounts/views.py

```diff
@@ -48,7 +48,7 @@ class SignupView(GenericAPIView):
     def get(self, request):
         # FIXME: Do some password checking, user validating, yada, yada
         form = UserSignupForm()
-        return Response({'form': form}, template_name='signup.html')
+        return Response({'form': form}, template_name='_signup.html')
```

Comment:
So HTML files that have an underscore at the start of their names are called partials. These are files that offer pieces of HTML that you can embed in other files.

In this case, the template should be `users/new.html` and that file would have the signup partial included.